src/components/whats-new.vue

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <modal :state="isVisible" backdrop :hideable="true" @hide="hideModal">
+    <template #title>{{ $t('title') }}</template>
+    <template #body>
+      <div class="modal-introduction">
+        {{ $t('body') }}
+      </div>
+      <div class="modal-actions">
+        <button type="button" class="btn btn-primary"
+          @click="hideModal">
+          {{ $t('action.gotIt') }}
+        </button>
+      </div>
+    </template>
+  </modal>
+</template>
+
+<script setup>
+import { ref, watch } from 'vue';
+
+import Modal from './modal.vue';
+
+import { useRequestData } from '../request-data';
+
+const { currentUser, projects } = useRequestData();
+const isVisible = ref(false);
+
+defineOptions({
+  name: 'WhatsNew'
+});
+
+watch(() => projects.dataExists, () => {
+  const canUpdateForm = currentUser.can('form.update') ||
+    projects.data.some(project => project.verbs.has('form.update'));
+  if (canUpdateForm && !currentUser.preferences.site.whatsNewDismissed2025_1) {
+    isVisible.value = true;
```

Comment:
Given that the modal isn't shown until after the projects response is received, and we know that response can be a little slow, I think it'd be a good idea to account for the case where another modal is already open. That modal could be `ProjectNew` or `AnalyticsIntroduction` or (in the future) the modal for getodk/central#849.

One way to check whether another modal is open is to see whether `document.body` has the `modal-open` class. I don't love using an HTML class like that, but it would probably work for now. I've had the thought multiple times that at some point, it could be helpful to store a piece of state in the `container` object that indicates whether any modal is open.